frontend/notification/notificationService.js

```diff
@@ -53,8 +53,8 @@
 
         service.formatMessage = function formatMessage(notification) {
             var entity_type = notification.entity_type;
-            var mainInst = notification.from.institution_name;
-            var otherInst = notification.to.institution_name;
+            var mainInst = notification.entity.institution_name || notification.from.institution_name;
```

Comment:
insert a comment explaining what is new notification data and what is old notification data